airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py

```diff
@@ -70,11 +70,17 @@ def records_with_state(records, state, stream_mapping, state_cursor_paths) -> It
         cursor_field = helper.field(stream.cursor_field)
         record_value = cursor_field.parse(record=record.record.data)
         try:
+            if state[stream_name] is None:
```

Comment:
is this needed? I'd expect the test to only run for streams that exist

Reply:
For some reason yes. It's annoying, but w/o it we end up with a TypeErro when we parse the None type. This was a real state coming from our SAT tests from hubspot:
```
type=<Type.STATE: 'STATE'> log=None spec=None connectionStatus=None catalog=None record=None state=AirbyteStateMessage(data={'companies': {'updatedAt': '2022-05-02T11:15:24.476000+00:00'}, 'contact_lists': {'updatedAt': 1634044407353}, 'contacts': {'updatedAt': '2022-05-05T14:31:15.068000+00:00'}, 'deals': {'updatedAt': '2022-05-11T05:39:31.131000+00:00'}, 'email_events': None})

```